deploy/operator/utils.sh

```diff
@@ -0,0 +1,46 @@
+function wait_for_crd() {
+  crd="$1"
+  namespace="$2"
+  echo "Waiting for CRD (${crd}) on namespace (${namespace}) to be defined..."
+  for i in {1..40}; do
+    oc get "crd/${crd}" -n "${namespace}" && break || sleep 10
```

Comment:
It should have the same `& grep -ivE "(no resources found|not found)"` like the pod

Reply:
nope. this will actually return a non-zero value
it's the ``--selector`` argument that makes ``oc get`` command return a list